### PR TITLE
Scenario Authoring documentation

### PR DESCRIPTION
**Pull request by arkadius**

### CODE REVIEW

File: docs/designingProcesses/BasicNodes.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
+# Introduction
+
+Node works with a data stream. It can produce, fetch, send, collect data or organize data flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.
```
Comment (mproch) [inline docs/designingProcesses/BasicNodes.md:3]: I think `description` is self explanatory?

File: docs/designingProcesses/BasicNodes.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
+# Introduction
+
+Node works with a data stream. It can produce, fetch, send, collect data or organize data flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.  
+
+With the exception of Source and Sink nodes (link) all other nodes have input and at least one output flow.
```
Comment (mproch) [inline docs/designingProcesses/BasicNodes.md:5]: this is not quite true, some custom transformers and processors (not available in generic distribution) can end scenario. Maybe sth like "most of the nodes (with the exception of e.g. Source and Sink)"...

File: docs/designingProcesses/BasicNodes.md [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+
+Some nodes can be disabled by selecting `Disable` checkbox.
```
Comment (mproch) [inline docs/designingProcesses/BasicNodes.md:7]: I think that basically these are filters and sinks

File: docs/designingProcesses/BasicNodes.md [hunk 4]
```diff
@@ -9,94 +17,79 @@ Records from the `source` which meet filter's condition go to the `true sink`, a
...
-### Flow
-Filter has one input and one or two outputs. 
-
-If there is one output, only outgoing pipe is named `true`, and each record which expression evaluates to true passes. Otherwise record is gone.
-
-If there are two outputs, one pipe is named `true` and another `false`. Each record which expression evaluates to `true` goes to `true` pipe, and other record goes to `false` pipe.  
-
-
-
-# Split
```
Comment (mproch) [inline docs/designingProcesses/BasicNodes.md:29]: I wonder, maybe it's not bad to have a mention that split enables parallel processing of many branches?

File: docs/designingProcesses/BasicNodes.md [hunk 5]
```diff
@@ -9,94 +17,79 @@ Records from the `source` which meet filter's condition go to the `true sink`, a
...
 ![switch_edge_default](../img/switch_edge_default.png)
 
 There can be at most one edge of type `Default`, and it gets all records that don't match any `Condition` edge. 
 
+
 ### Flow
 For each incoming record `Expression` is evaluated and result is assigned to `exprVal` variable. After that, records are tested against condition `Expressions` from output edges one by one. Record goes to first output which condition it satisfies. If record doesn't match any conditional output, and default output exists, record goes to default output. Otherwise mismatching record is filtered out.
 
 
+## SQL Variable
```
Comment (mproch) [inline docs/designingProcesses/BasicNodes.md:49]: I think it's a bit involved what this node does... I wonder if it's better to hide/delete it...

File: docs/designingProcesses/Spel.md [hunk 6]
```diff
@@ -131,11 +132,17 @@ There are a few notable exceptions:
...
+| `true && false` | false	| boolean |
+| `true OR false` | true	| boolean |
+| `true OR false` | true	| boolean |
 | `2 > 1 ? 'a' : 'b'` |	"a"	| String |
-| `2 < 1 ? 'a' : 'b'` |	"b"	String |
-| `#nonNullVar == null ? 'Success'` : 'Unknown' |	"Success" | String |
-| `#nullVar == null ? 'Success'` : 'Unknown' | "Unknown" | String |
-| `#nullVar?:'Unknown` | "Unknown" | String |
+| `2 < 1 ? 'a' : 'b'` |	"b"	| String |
+| `#nonNullVar == null ? 'Uknown' : 'Success'` |	"Success" | String |
```
Comment (mproch) [inline docs/designingProcesses/Spel.md:143]: Unknown?

File: docs/designingProcesses/BasicNodes.md (hunk 5 above)
Comment (ZbyszekMM) [reply docs/designingProcesses/BasicNodes.md:49]: It is such an important component, I would not.

**Review by mproch — APPROVED** [review]

**Pull request merged**